docs/api/field_type_reference.md

```diff
@@ -1501,6 +1501,165 @@ new FieldType\Value([
 ]);
 ```
 
+### GraphQL Field Type operations
+
+To get a Field of the Matrix Field Type with GraphQL, you will need to specify a Content ID, a Content Type, and a Field Type. 
+
+The types that are used for input and returned are named after the Type and the Field: 
+
+- `{TypeIdentifier}{FieldIdentifier}RowInput` e.g. `dish.nutritionFacts`, `event.agenda`
+- `{TypeIdentifier}{FieldIdentifier}Row`
+
+The example below shows a GraphQL query for a Recipe Content item (belonging to a Content Type with a Matrix Field added), that has two Fields:
+
+- `name`: `ezstring`
+- `ingredients`: `ezmatrix` with two columns: `ingredient` and `quantity`
+
+```
+{
+  content {
+    recipe(id: 123) {
+      name
+      ingredients {
+        ingredient
+        quantity
+      }
+    }
+  }
+}
+```
+
+The Type returned for the Matrix Field exposes columns defined in the Field definition:
+
+```
+{
+  "data": {
+    "content": {
+      "recipe": {
+        "name": "Cake ingredients",
+        "ingredients": [
+          {
+            "ingredient": "Butter",
+            "quantity": "200 grams"
+          },
+          {
+            "ingredient": "Sugar",
+            "quantity": "100 grams"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+#### Query for the Field Type and Field definition's details
+
+With this query you can inspect:
+
+- the existing types 
+- details of Content Types, and their Fields in the context of developing your own application
```

Comment:
One thing that I mentioned earlier: this sentence is more of a general info about GraphQL, Content types and fields definitions. It is not limited to the matrix field, and would deserve to be mentioned in a general GraphQL chapter.